packages/github-actions/teams-toolkit-cicd/src/base-error.ts

```diff
@@ -0,0 +1,36 @@
+// eslint-disable-next-line no-shadow
```

Comment:
Yeah, it comes from the official template. The code is expected to be put into a separated github repo in the end and follow the best practices of GitHub Action. It will not follow the mono repo's style.